homeassistant/components/nest/quality_scale.yaml

```diff
@@ -0,0 +1,77 @@
+rules:
+  # Bronze
+  config-flow:
+    status: todo
+    comment: Some fields are missing a data_description
+  brands: done
+  dependency-transparency: done
+  common-modules:
+    status: exempt
+    comment: The integration does not have a base entity or coordinator.
+  has-entity-name: done
+  action-setup:
+    status: exempt
+    comment: The integration does not register actions.
+  appropriate-polling:
+    status: exempt
+    comment: The integration does not poll.
+  test-before-configure:
+    status: todo
+    comment: |
+      The integration does a connection test in the configuration flow, however
+      it does not fail if the user has ipv6 misconfigured.
+  entity-event-setup: done
+  unique-config-entry: done
+  entity-unique-id: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  test-before-setup:
+    status: todo
+    comment: |
+      The integration does tests on setup, however the most common issues
+      observed are related to ipv6 misconfigurations and the error messages
+      are not self explanatory and can be improved.
+  docs-high-level-description: done
+  config-flow-test-coverage: done
```

Comment:
Ideally all the tests end in either a CREATE_ENTRY or ABORT so we also test that the flow is able to finish to avoid future race conditions

Reply:
Added as a comment.